src/utils.js

```diff
@@ -154,3 +154,16 @@ export function mapToCssModules(className, cssModule) {
   if (!cssModule) return className;
   return className.split(' ').map(c => cssModule[c] || c).join(' ');
 }
+
+/**
+ * Returns a new object with the key/value pairs from `obj` that are not in the array `omitKeys`.
+ */
+export function omit(obj, omitKeys) {
+  const result = {};
+  Object.keys(obj).forEach(key => {
```

Comment:
I was thinking of potential of `obj` to be null, but after a second I agree that it is unnecessary.